main/OpenCover.Framework/OpenCover.Framework.csproj

```diff
@@ -52,13 +52,11 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\Autofac.Configuration.3.3.0\lib\net40\Autofac.Configuration.dll</HintPath>
     </Reference>
-    <Reference Include="Gendarme.Framework, Version=2.11.0.0, Culture=neutral, PublicKeyToken=b07ccf3a9f1ab3c9, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\tools\GendarmeSigned\Gendarme.Framework.dll</HintPath>
+    <Reference Include="Gendarme.Framework">

```

Comment:
Gendarme comes unsigned so to link to them we need to create signed versions 

if you compile on the command line `build` or `build create-release` these files should be created and then be linked.